x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearning.java

```diff
@@ -442,9 +442,23 @@
     // This is for performance testing.  It's not exposed to the end user.
     // Recompile if you want to compare performance with C++ tokenization.
     public static final boolean CATEGORIZATION_TOKENIZATION_IN_JAVA = true;
-
-    public static final LicensedFeature.Persistent ML_JOBS_FEATURE =
-        LicensedFeature.persistent("machine-learning", "anomaly-detection-job", License.OperationMode.PLATINUM);
+    public static final String ML_FEATURE_FAMILY = "machine_learning";
+
+    public static final LicensedFeature.Persistent ML_JOBS_FEATURE = LicensedFeature.persistent(
```

Comment:
You may note that there isn't one for datafeeds. Tracking anomaly jobs is sufficient for now as there is a 1-1 mapping between datafeeds and anomaly jobs. Also, datafeeds are co-located with jobs on the same node.

Reply:
For consistency, can you rename the constant name to include `ANOMALY_DETECTION`? I should have done that when adjusting the name in the last PR.